invenio_theme/assets/semantic-ui/less/invenio_theme/theme/elements/container.overrides

```diff
@@ -1,6 +1,6 @@
-.ui.container.cover-page{
+.ui.container.cover-page {
   background-color: @brandColor;
-  margin-bottom: 3*@defaultMargin;
+  margin-bottom: 3*@coverPageDefaultMargin;
```

Comment:
Why 3*? should it be the 3 in the variable? That way we dont force users to do math :)

Reply:
It was like that :) Actually, you are right. I will do the `3*...` when I define the variable so people can override it completely :)